src/modules/navigator/GeofenceBreachAvoidance/geofence_breach_avoidance.cpp

```diff
@@ -182,7 +197,7 @@ GeofenceBreachAvoidance::generateLoiterPointForMultirotor(geofence_violation_typ
 
 	} else if (violation_type.flags.dist_to_home_exceeded) {
 
-		return waypointFromHomeToTestPointAtDist(math::max(_max_hor_dist_home - _min_hor_dist_to_fence_mc, 0.0f));
+		return waypointFromHomeToTestPointAtDist((computeBrakingDistanceMultirotor()/2) + _max_hor_dist_home);
```

Comment:
This is where the Hold point is computed. It originally would be set to the Geofence radius (_max_hor_dist_home) - a minimum distance that the drone could begin braking from based on acceleration/jerk limit parameters (_min_hor_dist_to_fence_mc). 
The intention was to ensure HOLD could not be outside Geofence.

Instead, we want to trigger a breach at Geofence but set Hold to the braking distance which is outside the Geofence to preserve the controlled deceleration and prevent overshoot corrections.

The braking distance was arbitrarily divided by 2 as it was still overkill.
(was overshooting by ~30m on a 30m geofence, i.e holding 60m from home)